### PR TITLE
[NEXUS-8215] testsuite spring cleaning

### PR DESCRIPTION
**Pull request by mcculls**
https://issues.sonatype.org/browse/NEXUS-8215
http://bamboo.s/browse/NX3-OSSF101-6 :white_check_mark:

### CODE REVIEW

File: plugins/security/nexus-kenai-plugin/src/test/java/org/sonatype/nexus/kenai/internal/KenaiClearCacheTest.java [hunk 1]
```diff
@@ -53,6 +48,13 @@ protected void setUp()
     securitySystem = lookup(SecuritySystem.class);
   }
 
+  @Override
+  protected void tearDown()
+      throws Exception
+  {
```
Comment (jdillon) [inline plugins/security/nexus-kenai-plugin/src/test/java/org/sonatype/nexus/kenai/internal/KenaiClearCacheTest.java]: minor watch this old-style formatting of simple methods that eat 3 lines.

**Comment by jdillon** [comment]
so many deletes, I love it

+1

**Comment by cstamas** [comment]
+1

**Comment by jdillon** [comment]
still +1

**Pull request closed without merging**